community/modules/remote-desktop/chrome-remote-desktop/variables.tf

```diff
@@ -86,11 +86,25 @@ variable "auto_delete_boot_disk" {
 }
 
 variable "name_prefix" {
-  description = "Name Prefix"
+  description = <<-EOT
+    An optional name for all VM and disk resources. 
+    If not supplied, `deployment_name` will be used. 
+    When `name_prefix` is supplied, and `add_deployment_name_before_prefix` is set, 
+    then resources are named by "<`deployment_name`>-<`name_prefix`>-<#>".
```

Comment:
What happens when "name_prefix" is `null` and "add_deployment_name_before_prefix" is set to `true`?

Reply:
The logic is [here](https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/4ce0ee50fce437c32e19b058db82525314a8497c/modules/compute/vm-instance/main.tf#L30). It does what you might expect `<deployment name>-0

This is the same interface and set of defaults from vm-instance. There was a lot of debate about it at the time it went in for vm-instance. I think it was decided it was the worst best option.